scrapers/intelli.py

```diff
@@ -51,7 +52,8 @@ def scrape_intelli():
         # region + country
         try:
             region = coffee_soup.find(text='Country').next_element.string 
-        except:
+        except AttributeError:
+            # check if it's a blend
             if 'Blend' in blend_or_origin:
```

Comment:
because it's only sometimes that a blend coffee doesn't have a region listed, currently none of the intelli blend coffees go into the except